app/queries/update_user_mutator.rb

```diff
@@ -82,6 +83,6 @@ def authorized?
   def user_params
     preferences = current_user.preferences
     preferences[:daily_digest] = daily_digest
-    { name: name, about: about, locale: locale, preferences: preferences }
+    { name: name, preferred_name: preferred_name.presence, about: about, locale: locale, preferences: preferences }
```

Comment:
You can avoid checking the presence of `preferred_name` by adding the key to `normalize_attribute` list in `User` model